README.md

```diff
@@ -47,129 +54,133 @@ For the assembly, JLCPCB did not have `TCR2EF19`, which I replaced it with `TLV7
+| Adapter - BTU bottom     | [Adapter BTU Bottom](https://github.com/Bastardkb/Charybdis/blob/d0e20cc/files/mods/btu/adapter_btu_bottom_v32.stl)                   |                     |
+| Printable 2.5mm BTU Ball | [Printable 2.5mm BTU Ball](https://github.com/Bastardkb/Charybdis/blob/322faad/files/mods/printable-btu/printable_btu_2.5mm_ball.stl) | You need 3 of those |
 
-As I mentioned above, I ordered with 0.8mm thickness and did not change anything else (other than dying it to black at no extra cost on 0.8mm)
 
+### Observations after build
+- I personally find the tents too high and could not find a good position to make use of them
+- I would like to replace the plates with metal ones to add more weight, and also less flexibility as ribbon wires add pressure
 
-### Flexible PCB for the plate (3x6)
-[Link to repo](https://github.com/Bastardkb/TBK-Mini-PCB-plate/releases/tag/2.21)
+## Electronic components
 
-Again, I ordered with 0.8mm thickness and dyed it black
+In this section, I will go through each component I have bought, and also give example links. As of writing I locate in Netherlands, therefore I tried to source some of the parts across European countries. 
 
-### 3D Prints
-A good friend and a colleague of mine [pbacterio](https://github.com/pbacterio) assisted me with 3D prints. I will add the link of each part  have 3D printed for a 3x6 build. This is different from the official README, as it does not use bearings for trackball and uses ceramic bearing balls. Some of the designs require the 3D model to be mirrored, I will link my mirrored files in this repository (Please refer to [Disclaimer](#disclaimer)).
+This list is based on the electronics BOM present on the Charybdis repo as of writing, so double-check there that you're not missing anything.
 
-- [Right case 3MF](https://github.com/Bastardkb/Charybdis/blob/2dad0ca/files/3x6%20mini/CMini_v1_v11.3mf), [STEP file](https://github.com/Bastardkb/Charybdis/blob/9afdbc9/files/3x6%20mini/CMini_v1_v11.step)
-- [Left case STL](./3d-prints/case_v3_v29_mirrorred.stl)
-- [Right Plate STL](https://github.com/Bastardkb/Charybdis/blob/3908164/files/3x6%20mini/plate_v1_v11.stl)
-- [Left Plate STL](./3d-prints/plate_v3_v29_mirrorred.stl)
-- [Adapter - top STL](https://github.com/Bastardkb/Charybdis/blob/c818c76/files/3x5%20nano/adapter_v2_top_v75.stl)
-- [Adapter BTU Bottom](https://github.com/Bastardkb/Charybdis/blob/d0e20cc/files/mods/btu/adapter_btu_bottom_v32.stl)
-- [Printable 2.5mm BTU Ball](https://github.com/Bastardkb/Charybdis/blob/322faad/files/mods/printable-btu/printable_btu_2.5mm_ball.stl), You need 3 of these
-- [Tent Left STL (Optional)](https://github.com/Bastardkb/Charybdis/blob/4924527/files/3x6%20mini/tent_v2_v57.3mf)
-- [Tent Right STL (Optional)](./3d-prints/tent_v2_v57_mirrorred.stl)
-- [Sensor Cover](https://github.com/Bastardkb/Charybdis/blob/9130a58/files/sensor_cover_v51.stl)
```

Comment:
Links refer to commits directly and not to a branch, that means mirrored files also refer to the same commit. As I mentioned above, I prepared this doc for myself also and I did not use original parts used such as BTU ball, I would rather keep this part as is, including the files. Let me know if you have objection.